### ISSUE TITLE
phonon property calculation error for `vasp type`

### ISSUE THREAD

**Issue by chunweizhu** [deepmodeling/APEX#50]
My calculation steps are as follows

**1.relaxation**
~~~sh
apex test relaxation.json make_relax
apex test relaxation.json run_relax -m machine.json
apex test relaxation.json post_relax 
~~~
**2.phonon property calculation**
~~~sh
apex test property.json make_props
apex test property.json run_props -m machine.json  
apex test property.json post_props # An error occurred !
~~~

**error:**
~~~sh
 File "/share/home/user/.conda/envs/deepmd222/bin/apex", line 8, in <module>
    sys.exit(main())
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/main.py", line 115, in main
    run_step(
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/run_step.py", line 59, in run_step
    post_property(structures, inter_parameter, param)
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/common_prop.py", line 255, in post_property
    prop.compute(
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/property/Property.py", line 103, in compute
    res, ptr = self._compute_lower(output_file, task_dirs, all_res)
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/property/Phonon.py", line 387, in _compute_lower
    shutil.copyfile("task.000000/POSCAR-unitcell", "POSCAR-unitcell")
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/shutil.py", line 234, in copyfile
    raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
shutil.SameFileError: 'task.000000/POSCAR-unitcell' and 'POSCAR-unitcell' are the same file
~~~

**version:**
APEX v1.1.1

**Comment by ZLI-afk** [comment]
Could you please share your `property.json`? And btw, have you try to delete  `phonon_00/POSCAR-unitcell` manually and then do the `post_props` to see if it works?

**Comment by ZLI-afk** [comment]
Thanks for your report. This issue has been identified and fixed with newest version of `apex-flow==1.1.2`. You may need to run `pip3 install apex-flow -U` to update `APEX` to `1.1.2`, then try to do the `post` again.

**Comment by chunweizhu** [comment]
Thank you for reply, but a new problem has arisen, which seems to be caused by `apex test property.json run_props -m machine.json `not take back the `vasprun.xml` file.

**error**
~~~sh
(deepmd222) [user@sonmi apextest]$ apex test property.json post_props
---------------------------------------------------------------
░░░░░░█▐▓▓░████▄▄▄█▀▄▓▓▓▌█░░░░░░░░░░█▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀█░░░░░
░░░░░▄█▌▀▄▓▓▄▄▄▄▀▀▀▄▓▓▓▓▓▌█░░░░░░░░░█░░░░░░░░▓░░▓░░░░░░░░█░░░░░
░░░▄█▀▀▄▓█▓▓▓▓▓▓▓▓▓▓▓▓▀░▓▌█░░░░░░░░░█░░░▓░░░░░░░░░▄▄░▓░░░█░▄▄░░
░░█▀▄▓▓▓███▓▓▓███▓▓▓▄░░▄▓▐██░░░▄▀▀▄▄█░░░░░░░▓░░░░█░░▀▄▄▄▄▄▀░░█░
░█▌▓▓▓▀▀▓▓▓▓███▓▓▓▓▓▓▓▄▀▓▓▐█░░░█░░░░█░░░░░░░░░░░░█░░░░░░░░░░░█░
▐█▐██▐░▄▓▓▓▓▓▀▄░▀▓▓▓▓▓▓▓▓▓▌█░░░░▀▀▄▄█░░░░░▓░░░▓░█░░░█▒░░░░█▒░░█
█▌███▓▓▓▓▓▓▓▓▐░░▄▓▓███▓▓▓▄▀▐█░░░░░░░█░░▓░░░░▓░░░█░░░░░░░▀░░░░░█
█▐█▓▀░░▀▓▓▓▓▓▓▓▓▓██████▓▓▓▓▐█░░░░░▄▄█░░░░▓░░░░░░░█░░█▄▄█▄▄█░░█░
▌▓▄▌▀░▀░▐▀█▄▓▓██████████▓▓▓▌██░░░█░░░█▄▄▄▄▄▄▄▄▄▄█░█▄▄▄▄▄▄▄▄▄█░░
▌▓▓▓▄▄▀▀▓▓▓▀▓▓▓▓▓▓▓▓█▓█▓█▓▓▌██░░░█▄▄█░░█▄▄█░░░░░░█▄▄█░░█▄▄█░░░░
█▐▓▓▓▓▓▓▄▄▄▓▓▓▓▓▓█▓█▓█▓█▓▓▓▐█░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
---------------------------------------------------------------
        AAAAA         PPPPPPPPP     EEEEEEEEEE  XXX       XXX
       AAA AAA        PPP     PPP   EEE           XXX   XXX
      AAA   AAA       PPP     PPP   EEE            XXX XXX
     AAAAAAAAAAA      PPPPPPPPP     EEEEEEEEE       XXXXX
    AAA       AAA     PPP           EEE            XXX XXX
   AAA         AAA    PPP           EEE           XXX   XXX
  AAA           AAA   PPP           EEEEEEEEEE  XXX       XXX
---------------------------------------------------------------
==>> Alloy Property EXplorer using simulations (v1.1.3)
Checking input files...
-------Singel step local debug mode--------
Posting property results locally...
        _
  _ __ | |__   ___  _ __   ___   _ __  _   _
 | '_ \| '_ \ / _ \| '_ \ / _ \ | '_ \| | | |
 | |_) | | | | (_) | | | | (_) || |_) | |_| |
 | .__/|_| |_|\___/|_| |_|\___(_) .__/ \__, |
 |_|                            |_|    |___/
                                      2.21.0

Python version 3.10.12
Spglib version 2.0.2


"task.0*/vasprun.xml" was not found.
  ___ _ __ _ __ ___  _ __
 / _ \ '__| '__/ _ \| '__|
|  __/ |  | | | (_) | |
 \___|_|  |_|  \___/|_|

FORCE_SETS can not be created
        _
  _ __ | |__   ___  _ __   ___   _ __  _   _
 | '_ \| '_ \ / _ \| '_ \ / _ \ | '_ \| | | |
 | |_) | | | | (_) | | | | (_) || |_) | |_| |
 | .__/|_| |_|\___/|_| |_|\___(_) .__/ \__, |
 |_|                            |_|    |___/
                                      2.21.0

Python version 3.10.12
Spglib version 2.0.2

"band.conf" was read as phonopy configuration file.
Crystal structure was read from "POSCAR-unitcell".
Unit of length: angstrom
Band structure mode
Settings:
  Supercell: [2 1 1]
Spacegroup: Pmma (51)
Use -v option to watch primitive cell, unit cell, and supercell structures.


"FORCE_SETS" was not found.
  ___ _ __ _ __ ___  _ __
 / _ \ '__| '__/ _ \| '__|
|  __/ |  | | | (_) | |
 \___|_|  |_|  \___/|_|

Traceback (most recent call last):
  File "/share/home/user/.conda/envs/deepmd222/bin/phonopy-bandplot", line 779, in <module>
    main(get_options())
  File "/share/home/user/.conda/envs/deepmd222/bin/phonopy-bandplot", line 764, in main
    _write_gnuplot_data(args)
  File "/share/home/user/.conda/envs/deepmd222/bin/phonopy-bandplot", line 733, in _write_gnuplot_data
    bands_data = [_read_band_yaml(fname) for fname in filenames]
  File "/share/home/user/.conda/envs/deepmd222/bin/phonopy-bandplot", line 733, in <listcomp>
    bands_data = [_read_band_yaml(fname) for fname in filenames]
  File "/share/home/user/.conda/envs/deepmd222/bin/phonopy-bandplot", line 194, in _read_band_yaml
    with open(filename, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'band.yaml'
Traceback (most recent call last):
  File "/share/home/user/.conda/envs/deepmd222/bin/apex", line 8, in <module>
    sys.exit(main())
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/main.py", line 115, in main
    run_step(
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/run_step.py", line 59, in run_step
    post_property(structures, inter_parameter, param)
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/common_prop.py", line 255, in post_property
    prop.compute(
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/property/Property.py", line 103, in compute
    res, ptr = self._compute_lower(output_file, task_dirs, all_res)
  File "/share/home/user/.conda/envs/deepmd222/lib/python3.10/site-packages/apex/core/property/Phonon.py", line 442, in _compute_lower
    result_points = ptr_data.split('\n')[1][4:]
IndexError: list index out of range
~~~

**Comment by ZLI-afk** [comment]
Problem is fixed in version `2.1.4`. Please try to upgrade. Thanks.

**Comment by chunweizhu** [comment]
Thank you for your patient reply, this works for me. 😊
